components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -303,6 +303,28 @@ Chef Infra Server lets you delete the existing clients one at a time. To delete
 
 {{< figure src="/images/automate/delete-a-client.png" alt="Delete a Client">}}
 
+### Nodes
+
+A [Node]({{< relref "nodes" >}}) is a communication between the Infra Chef Server and the Infra Clients. Nodes are used when an Infra Client asks for a policy from the Infra Chef Server.
```

Comment:
Nodes are the devices managed by Chef Infra. https://docs.chef.io/nodes/. The communication between Chef Infra Client and Chef Infra Server happens as part of the Chef Infra Client Run. Chef Infra Client pulls down the node object from Chef Infra Server and the Policyfile is part of this. https://docs.chef.io/chef_client_overview/

There was a word order problem in the original:
```suggestion
A [Node]({{< relref "nodes" >}}) is a communication between the Chef Infra Server and the Infra Clients. Nodes are used when an Infra Client asks for a policy from the Infra Chef Server.
```